frontend/yarn-error.log

```diff
@@ -0,0 +1,54 @@
+Arguments: 
```

Comment:
I don't remember if the project really needs this file. I think we can put it on .gitignore

Reply:
Good catch! Removed ✅